gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,81 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data
+
+    def initialize
+      self.appliance_pg_data = ENV.fetch('APPLIANCE_PG_DATA')
+      set_defaults
+    end
+
+    def set_defaults
+      self.cluster_name              = "replication_cluster"
+      self.node_number               = 1
+      self.primary_database          = "vmdb_production"
+      self.primary_database_username = "root"
+      self.primary_database_password = nil
+      self.primary_host              = nil
+      self.standby_host = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Replication Standby Server\n")
+      ask_for_cluster_name
+      ask_for_node_number
+      ask_for_primary_database_credentials
+      ask_for_standby_host
+      confirm(:including_standby_host)
+    end
+
+    def ask_for_standby_host
+      ask_for_ip_or_hostname("Standby Server hostname or IP address", standby_host)
+    end
+
+    def activate
+      say("Configuring Replication Standby Server...")
+      confirm_no_standby_database? &&
+        already_configured? &&
+        create_config_file(standby_host) &&
+        clone_standby_server &&
+        start_postgres &&
+        register_standby_server
+    end
+
+    def confirm_no_standby_database?
```

Comment:
This method name seems strange. It isn't really a predicate right? Maybe `data_dir_empty?` would be better?